src/client/components/checkout/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { Button, ButtonProps, LinkButton } from '../buttons'
+import { PaymentInfo } from './PaymentInfo'
+
+const { white, purple500, gray900 } = colorsV3
+
+export type Props = {
+  buttonText: string
+  buttonOnClick?: () => void
+  buttonLinkTo?: string
+}
+type FooterButtonProps = {
+  background: ButtonProps['background']
+  foreground: ButtonProps['foreground']
+}
+
+const Wrapper = styled.div`
+  width: 100vw;
+  height: 5rem;
+  padding: 1rem;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  background-color: ${white};
+  box-shadow: 0px -4px 8px rgba(0, 0, 0, 0.05),
+    0px -8px 16px rgba(0, 0, 0, 0.05);
+`
+const InnerWrapper = styled.div`
+  width: 100%;
+  max-width: 628px;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+export const Footer = ({ buttonText, buttonOnClick, buttonLinkTo }: Props) => {
+  const buttonProps: FooterButtonProps = {
+    background: purple500,
+    foreground: gray900,
+  }
+
+  return (
+    <Wrapper>
+      <InnerWrapper>
+        <PaymentInfo />
+        {buttonText && (
```

Comment:
[Discussion]

What do you think about an API like this?
```
type Props =  { actionButton: JSX.Element }

export const Footer = ({ actionButton }: Props) => {
   return (
     <Wrapper>
       <InnerWrapper>
         <PaymentInfo />
         {actionButton}
       </InnerWrapper>
     </Wrapper>
   )
}
```
I think it's easier to maintain since are just compose `Footer` with something that needs to be rendered: `actionButton`. With the current used approach, If you end needing to add more stuff specific to be button scenario, for example, you might need to accept to change `Footer` more button related props just so on pass them down, which end up making `Footer` more complex to work with IMHO.

Reply:
Hm, yes that's a good point. Probably won't do that remake now though, since I'm not sure it would be worth the effort. Or what do you think?

Reply:
Yeah, you're right.